src/api/eventListeners/query/get.ts

```diff
@@ -0,0 +1,151 @@
+import {
+  EventListenersListSchema,
+  EventListenersListSchemaType,
+} from "../schema";
+import {
+  QueryFunctionContext,
+  useQuery,
+  useQueryClient,
+} from "@tanstack/react-query";
+
+import { apiFactory } from "~/api/apiFactory";
+import { eventListenerKeys } from "..";
+import moment from "moment";
+import { useApiKey } from "~/util/store/apiKey";
+import { useNamespace } from "~/util/store/namespace";
+import { useStreaming } from "~/api/streaming";
+
+const updateCache = (
+  oldData: EventListenersListSchemaType | undefined,
+  message: EventListenersListSchemaType
+) => {
+  if (!oldData) {
+    return message;
+  }
+  /**
+   * TODO: copied from events. Does this behave the same?
```

Comment:
Fixed in 6eb9f631c8a686c25b79b05961354646677cfb90

You're right, removed TODO and updated "logs" to "results". I copied the comment over with that method and thought it can't hurt to keep it.